tests/aggregation_regress/tests.py

```diff
@@ -1523,6 +1524,25 @@ class DistinctAggregate(Aggregate):
             allow_distinct = True
         DistinctAggregate('foo', distinct=True)
 
+    def test_groupby_after_aggregation_and_subquery(self):
```

Comment:
Looks like this will need to be skipped on Oracle like the test added in https://github.com/django/django/commit/42c08ee46539ef44f8658ebb1cbefb408e0d03fe is.

Talking of this test we likely want to move this one right below it.

Reply:
Test added in 42c08ee46539ef44f8658ebb1cbefb408e0d03fe is not skipped on Oracle anymore, see 3a941230c85b2702a5e1cd97e17251ce21057efa.

The new tests crashes because you cannot use aliases in the `GROUP BY` clause no Oracle. With this patch (and before 42c08ee46539ef44f8658ebb1cbefb408e0d03fe) it raises:
```
django.db.utils.DatabaseError: ORA-00904: "TOP_RATING": invalid identifier
```
Without this patch (and after 42c08ee46539ef44f8658ebb1cbefb408e0d03fe) it raises:
```
django.db.utils.DatabaseError: ORA-22818: subquery expressions not allowed here
```
which is more appropriate and could be skipped with `@skipUnlessDBFeature('supports_subqueries_in_group_by')`.

Reply:
I've added `@skipUnlessDBFeature('supports_subqueries_in_group_by')` to the test.